app/controllers/tas_controller.rb

```diff
@@ -43,6 +43,18 @@ def update
     respond_with @role, location: course_tas_path(current_course)
   end
 
+  def destroy
+    @role = record
+    begin
+      @role.destroy!
+      flash_now(:success, I18n.t('flash.tas.destroy.success', user_name: @role.user_name))
```

Comment:
put this line in an `else` block (look up the syntax for `else` with `begin`/`rescue`) as this indicates that the code should only run if no error was triggered

Also, this is a good time to use `head` to send an appropriate status code in the response. For DELETE, use `:ok` for a success, `:conflict` for the delete restriction error, and `:bad_request` for other errors